### PR TITLE
Fix configuration

### PR DESCRIPTION
**Pull request by AlexanderHagenhoff**
Fix some configuration and add extra application-vagrant.yml .. we need/want it anyways 

@rebuy-de/prp-rebuy-silo-archetype 
@rebuy-de/it-outbound

### CODE REVIEW

**Comment by JanZieglerBerlin** [comment]
👍

File: src/main/resources/archetype-resources/silo/src/main/resources/application-vagrant.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+stage: vagrant
+
+spring:
+    datasource:
+        url: jdbc:CHANGEME://${spring.config.name}.CHANGME.service.vagrant.consul/CHANGEME
```
Comment (bjoernhaeuser) [inline src/main/resources/archetype-resources/silo/src/main/resources/application-vagrant.yml]: that can be removed, why did you add it? Normally the "vagrant" should be replaced by `${stage}`

File: src/main/resources/archetype-resources/silo/src/main/resources/application.yml [hunk 2]
```diff
@@ -40,9 +37,7 @@ permissions:
 remoteTokenServices:
     clientId: ${rootArtifactId}
     secret: CHANGEME
-    endpoint: http://customer-silo.vg/oauth/check_token
 
 consul:
     name: ${spring.config.name}
-    agent: 192.168.33.10
```
Comment (bjoernhaeuser) [inline src/main/resources/archetype-resources/silo/src/main/resources/application.yml]: that would mean you cannot start it without the profile, which sucks :(

**Comment by bjoernhaeuser** [comment]
I do not understand the motivation behind this.

**Comment by AlexanderHagenhoff** [comment]
discussed and closed
